content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,142 @@
+---
+title: AWS Lambda Getting Started with ASM
+kind: documentation
+is_beta: true
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: "/security/application_security/how-appsec-works/"
+      tag: "Documentation"
+      text: "How Application Security Works"
+    - link: "/security/default_rules/#cat-application-security"
+      tag: "Documentation"
+      text: "OOTB Application Security Management Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting Application Security Management"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "Application Threat Monitoring and Protection"
+---
+
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Lambda extension.</div>
+
+You can monitor your functions running in AWS Lambda with Datadog Application Security Management (ASM). See [Setup and Configure][4] for information about what ASM features are supported for serverless functions.
+
+In general, setting up ASM for AWS Lambda involves:
+
+1. Identifying functions that are vulnerable or are under attack, which would most benefit from ASM. Find them on [the Security tab of your Service Catalog][1].
+2. Set up ASM instrumentation via the serverless plugin or manually setting the ARN.
+3. Enabling the library to collect the application security data from the functions and send it to Datadog.
+4. Triggering security signals in your application and seeing how Datadog displays the resulting information.
+
+## Prerequisites
+
+- [Serverless APM][2] is configured on the Lambda function to send traces directly to Datadog (can't use X-Ray integration).
+
+## Get started
+
+{{< tabs >}}
+{{% tab "Serverless Framework" %}}
+
+The [Datadog Serverless plugin][1] automatically configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
+
+To install and configure the Datadog Serverless plugin:
+
+1. Install the Datadog Serverless plugin:
+   ```sh
+   serverless plugin install --name serverless-plugin-datadog
+   ```
+2. Enable ASM by updating your `serverless.yml` (or whichever way you set environment variables for your function):
+   ```yaml
+   environment:
+     DD_SERVERLESS_APPSEC_ENABLED: true
+   ```
+   For **Go functions only** also add:
+   ```yaml
+   DD_UNIVERSAL_INSTRUMENTATION:
+     environment:
+       DD_UNIVERSAL_INSTRUMENATION: true
+   ```
```

Comment:
Add a section for Node and Python environment variables:

For **Node or Python** functions only, please add:
```yaml
    environment:
        DD_EXPERIMENTAL_ENABLE_PROXY: true
```